treecraft/gui/sequence_dialog.py

```
@@ -0,0 +1,416 @@
+    """Dialog for adding new sequences"""
+    
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self.setWindowTitle("Add Sequence")
+        self.setMinimumWidth(500)
+        self.setMinimumHeight(300)
+        
+        self.sequence_id = ""
+        self.sequence = ""
+        
+    def __init__(self, parent=None):
```

Ruff (0.8.2)

Redefinition of unused `__init__` from line 12

(F811)